ecommerce/CartController.php

```diff
@@ -94,7 +94,7 @@ public function __construct( $register_hooks = true ) {
 			 *
 			 * @since 3.0.0
 			 */
-			add_action( 'wp_ajax_tutor_cart_delete', array( $this, 'tutor_cart_delete' ) );
+			add_action( 'wp_ajax_tutor_course_from_cart', array( $this, 'tutor_course_from_cart' ) );
```

Comment:
I think this`wp_ajax_tutor_course_from_cart` should be `wp_ajax_tutor_delete_course_from_cart `

Reply:
Updated.